yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -284,9 +284,10 @@ export class AztecNodeService implements AztecNode, AztecNodeAdmin, Traceable {
       options.prefilledPublicData,
       telemetry,
     );
-    const circuitVerifier = config.realProofs
-      ? await BBCircuitVerifier.new(config)
-      : new TestCircuitVerifier(config.proverTestVerificationDelayMs);
+    const circuitVerifier =
+      config.realProofs || config.debugForceTxProofVerification
+        ? await BBCircuitVerifier.new(config)
+        : new TestCircuitVerifier(config.proverTestVerificationDelayMs);
```

Comment:
I thought it would be safer to do it this way (where we verify Chonk proofs even though `PROVE_RREAL_PROOFS=false`) rather than disabling proof verification when `PROVER_REAL_PROOFS=true`